### PR TITLE
Feature/DTGB-433: 404 page

### PR DESCRIPTION
**Pull request by meirege**
## Screenshots (if appropriate):
image

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have updated the documentation accordingly.
- [ ] I have updated the style guide CHANGELOG accordingly.
- [ ] I have updated the gent_base CHANGELOG accordingly.

### CODE REVIEW

File: styleguide/components/41-organisms/help-block/help-block.twig [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+      'text': 'And a third link',
+      'modifier': 'standalone-link'
+    } %}
+    {% include '@paragraph' with {
+    'text': 'Donec condimentum morbi et mollis ultrices eleifend a montes ante ad morbi est ad eleifend ullamcorper. Sapien tristique lectus integer ac condimentum orci nunc quisque',
+    } %}
+  </div>
+
+  <div class="help-block-section">
+    {% include '@h3' with {'h3_text': ''} %}
```
Comment (delrueba) [inline styleguide/components/41-organisms/help-block/help-block.twig]: No empty headings

File: styleguide/components/41-organisms/help-block/help-block.twig [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+    {% include '@search' with {
+      'label': '2. Try to find the page through our search function',
```
Comment (delrueba) [inline styleguide/components/41-organisms/help-block/help-block.twig]: Label content does not match the input field.

**Review by delrueba — CHANGES_REQUESTED** [review]

File: styleguide/components/41-organisms/help-block/_help-block.scss [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+  &-section {
+    margin-bottom: 2rem;
+
+    &:last-of-type {
+      margin: 0;
+    }
+  }
+
+  .search {
+    width: span(7 of 9);
```
Comment (delrueba) [inline styleguide/components/41-organisms/help-block/_help-block.scss]: This gives strange results on mobile etc.
It's never full width, even though the rest of the block is.

File: styleguide/components/41-organisms/help-block/help-block.twig [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+
+  <div class="help-block-section">
+  {% include '@h3' with {'h3_text': '3. Continue to navigate from here'} %}
+  {% include '@paragraph' with {
+    'text': 'This can be done in several ways:',
+  } %}
+  {% include '@list' with {
+    'type': 'unordered',
+    'items': [
+      '<i class="icon-home"></i> Return to the <a href="javascript:(void);" alt="Homepage">homepage</a>',
```
Comment (delrueba) [inline styleguide/components/41-organisms/help-block/help-block.twig]: aria hidden

File: styleguide/components/41-organisms/help-block/help-block.twig [hunk 5]
```diff
@@ -0,0 +1,55 @@
...
+      '<i class="icon-fancyback"></i>Use the back button of your browser'
```
Comment (delrueba) [inline styleguide/components/41-organisms/help-block/help-block.twig]: aria-hidden

File: styleguide/components/41-organisms/search-block/_search-block.scss [hunk 6]
```diff
@@ -0,0 +1,71 @@
...
+        font-size: .8rem;
+      }
+    }
+
+    i {
+      position: absolute;
+      top: 0;
+      bottom: 0;
+      left: 0;
+      height: 2rem;
```
Comment (delrueba) [inline styleguide/components/41-organisms/search-block/_search-block.scss]: Don't mix em and rem
font-size is em, so height should be too.

**Review by delrueba — CHANGES_REQUESTED** [review]

File: styleguide/components/41-organisms/help-block/_help-block.scss (hunk 3 above)
Comment (meirege) [reply styleguide/components/41-organisms/help-block/_help-block.scss]: Forgot the breakpoint. :)

File: styleguide/components/41-organisms/help-block/help-block.twig (hunk 2 above)
Comment (meirege) [reply styleguide/components/41-organisms/help-block/help-block.twig]: Fixed.

File: styleguide/components/41-organisms/help-block/help-block.twig (hunk 1 above)
Comment (meirege) [reply styleguide/components/41-organisms/help-block/help-block.twig]: Fixed.

File: styleguide/components/41-organisms/help-block/help-block.twig (hunk 4 above)
Comment (meirege) [reply styleguide/components/41-organisms/help-block/help-block.twig]: Fixed.

File: styleguide/components/41-organisms/help-block/help-block.twig (hunk 5 above)
Comment (meirege) [reply styleguide/components/41-organisms/help-block/help-block.twig]: Fixed.

File: styleguide/components/41-organisms/search-block/_search-block.scss (hunk 6 above)
Comment (meirege) [reply styleguide/components/41-organisms/search-block/_search-block.scss]: Fixed.

**Review by delrueba — APPROVED** [review]

**Pull request merged**